### PR TITLE
fix: use Swagger integration to enable CORS

### PR DESCRIPTION
**Pull request by jasonwalsh**
This was a huge pain in the ass, but it _should_ fix all of the CORS-related issues.

**Testing**

The workflow is a bit manual, but in order to test, invoke the following commands in the Vagrant environment:

1. `env $(cat .env) make deploy` (this deploys the API Gateway)
2. Visit http://hopdrops.io
3. Choose locations and click **Next**

If all goes well, you should see the following screen:

Screen Shot 2019-10-21 at 10 12 21 PM

### CODE REVIEW

File: functions/accounts.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def handler(event, context):
     table.update_item(
         ExpressionAttributeValues={
             ':c': 'hopdrops',
-            ':l': locations
+            ':l': dumps(locations)
```
Comment (brh55) [inline functions/accounts.py]: Interesting, so this solves the float issues?

Comment (jasonwalsh) [reply functions/accounts.py]: yeah, it's not deal, but it prevents having to cast floating point values, especially deeply nested ones.

File: swagger.yaml [hunk 2]
```diff
@@ -44,6 +44,43 @@ info:
...
+              "statusCode": 200
+            }
+        responses:
+          default:
+            responseParameters:
+              method.response.header.Access-Control-Allow-Credentials: "'true'"
+              method.response.header.Access-Control-Allow-Headers: >-
+                'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'
+              method.response.header.Access-Control-Allow-Methods: "'OPTIONS,PATCH'"
+              method.response.header.Access-Control-Allow-Origin: "'http://localhost:8080'"
```
Comment (brh55) [inline swagger.yaml]: Should this be localhost or should it be the s3 bucket?

**Comment by brh55** [comment]
Overall it looks good

File: swagger.yaml (hunk 2 above)
Comment (jasonwalsh) [reply swagger.yaml]: I'm going to make it both

File: functions/accounts.py (hunk 1 above)
Comment (brh55) [reply functions/accounts.py]: 👍

**Pull request merged**